scripts/templates/create-page/EmptyModule.mustache

```diff
@@ -0,0 +1,20 @@
+// Import variables
+@import '../../../styles/variables';
+
+// Define your classNames using camel case (NO DASHES)
```

Comment:
I'd prefer not to have an SCSS file in the template. If you want to include a styling example, could you maybe use `mergeStyleSets` instead (to keep things simple)? (Or just let people who want styles copy from other existing files.)

Reply:
That's fair, and will simplify things a little.